apps/rush-lib/src/cli/actions/BaseRushAction.ts

```diff
@@ -32,14 +32,14 @@ export interface IBaseRushActionOptions extends ICommandLineActionOptions {
 }
 
 /**
- * The base Rush action that all Rush actions should extend.
+ * The base class for a few specialized Rush command-line actions that
+ * can be used without a rush.json configuration.
  */
-export abstract class BaseRushAction extends CommandLineAction {
+export abstract class BaseConfiglessRushAction extends CommandLineAction {
```

Comment:
>BaseConfiglessRushAction [](start = 22, length = 24)

`BaseConfigOptionalRushAction`?

Reply:
I find that grammar more ambiguous

---
In reply to: [217269841](https://github.com/Microsoft/web-build-tools/pull/823#discussion_r217269841) [](ancestors = 217269841)